### DISCUSSION TITLE
mkdocs for CleanerWorkflow problem

### DISCUSSION THREAD

**Discussion by buchmann-dhi** [DHI/python-package-development#17] [General]
I have a problem, when I try to use the docstring doc for CleanerWorkflow 
eg: `::: tscleaner.CleanerWorkflow`
Somehow, mkdocs does not find the Class (or doc), even though it is in the same file as the other cleaners.

```
(..) C:\..\timeseriescleaner>mkdocs serve
INFO    -  Building documentation...
INFO    -  Cleaning site directory
ERROR   -  mkdocstrings: tscleaner.CleanerWorkflow could not be found
ERROR   -  Error reading page 'reference.md':
ERROR   -  Could not collect 'tscleaner.CleanerWorkflow'

Aborted with a BuildError!
```

The reference is from an md-file on docs/ - the same file that successfully references the three @dataclasses.
Any ideas/hints?

**Comment by buchmann-dhi** [comment]
Resolved:
I just had to provide full name, eg `::: tscleaner.cleaning.CleanerWorkflow`. The first three cleaners are somehow exposed as eg `::: tscleaner.SpikeCleaner` (but `::: tscleaner.cleaning.SpikeCleaner` also works).

**Comment by jsmariegaard** [comment]
Depends on whether you imported them in you `__init__.py` or not (i.e. if they are available in the root namespace)

**Reply by buchmann-dhi** [reply]
Got it, thank you @jsmariegaard
